recipe/meta.yaml

```diff
@@ -1,40 +1,73 @@
-{% set version = "0.15rc1" %}
+{% set name = "movingpandas" %}
+{% set version = "0.17.2" %}
 
 package:
-  name: movingpandas
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/m/movingpandas/movingpandas-{{ version }}.tar.gz
-  sha256: 53b8eceaeec8a1c6f431874da6b63a3ca0ccca9a56c27ffd07c80a3b84194c1b
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a84bab2473a0ee254ac10bd562b4c4c955ea69820a033be0220abd4cd2476c6e
+  patches:
+    - patches/0001-fix-long-description-utf-8.patch  # [win]
 
 build:
   number: 0
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . -vv"
+  # fiona, rtree not available for s390x
+  skip: true  # [py<38 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
+  build:
+    - m2-patch    # [win]
   host:
-    - python >=3.7
+    - python
     - pip
+    - setuptools
+    - wheel
   run:
-    - python >=3.7
+    - python
     - fiona
     - geopandas-base
     - geopy
     - matplotlib-base
     - rtree
 
+# skip these tests because they depend on json files not in the tarball
+{% set tests_to_skip = "test_mf_file" %}
+{% set tests_to_skip = tests_to_skip + " or test_mf_collection_file" %}
+{% set tests_to_skip = tests_to_skip + " or test_mf_collection_needs_id" %}
+{% set tests_to_skip = tests_to_skip + " or test_wrong_property_raises_error" %}
+{% set tests_to_skip = tests_to_skip + " or test_gdf_to_mf_json" %}
+{% set tests_to_skip = tests_to_skip + " or test_outlier_cleaner" %}
+# skip this test on win, it requires tk
+{% set tests_to_skip = tests_to_skip + " or test_plot_does_not_alter_df" %}  # [win]
+# skip this test on win, it requires tcl
+{% set tests_to_skip = tests_to_skip + " or test_support_for_other_time_column_names" %}  # [win]
+{% set tests_to_skip = tests_to_skip + " or test_plot_exists" %}  # [win]
+{% set tests_to_skip = tests_to_skip + " or test_plot_speed_not_altering_collection" %}  # [win]
```

Comment:
Will these tests pass if we add `tk` to `test/requires`? Because tk is available on win64 https://github.com/AnacondaRecipes/tk-feedstock/blob/master/recipe/meta.yaml.
FYI, our `tk` package already contains `tcl`

Reply:
They will, they require `matplotlib` though, and if I directly add `matplotlib` in the tests without `tk/tcl` they will also pass. The tests do not have particular code added or dependencies to `tk/tcl`, and it's the same for `not win`.

It seems to me that for `win` we need to have `matplotlib` in `run` for `win`.

Reply:
thanks for the details, I really appreciate